lib/engines/content_block_manager/lib/tasks/delete_draft_content_blocks.rake

```diff
@@ -0,0 +1,12 @@
+namespace :content_block_manager do
+  desc "Delete draft Content Block Editions and any orphaned Content Block Documents"
+  task delete_draft_content_blocks: :environment do
+    draft_content_editions = ContentBlockManager::ContentBlock::Edition.where(state: "draft")
```

Comment:
I wonder if we should only fetch draft editions that are, say 2 days old? If we run this as it stands, we risk deleting in-progress applications. Something like:

```suggestion
    draft_content_editions = ContentBlockManager::ContentBlock::Edition.where(state: "draft").where("updated_at < ?", 2.days.ago)
```